### PR TITLE
Discard all shortcut

### LINKED ISSUE #7588: Add a Keyboard shortcut for "Discarding All Changes"
**Please describe the problem you think should be solved**
Add a Keyboard shortcut for "Discarding All Changes"

> “When I wish to discard all my local changes, it is a pain to bring my cursor to the branch menu and then down to the discard all changes.”

**[Optional] Do you have any potential solutions in mind?**
- Add a shortcut of `CmdOrCtrl + Shift + Delete`
- Add a shortcut of `CmdOrCtrl + Delete` and move delete repo to `CmdOrCtrl + Shift + Delete`

### PR DESCRIPTION
**Pull request by say25**
## Overview

**Closes #7588**

## Description

- Adds menu item shortcut to "Discard all Changes" (Cmd/Ctrl+Shift+Delete)
- Removes some extra const string declarations

## Release notes

Notes: Adds menu item shortcut to "Discard all Changes"

### CODE REVIEW

**Comment by tierninho** [comment]
@say25 Thanks for adding this shortcut -works as me 👍 

One clarification, did you mean to use "Delete-right" as it means the use of the function key is also required (at least for Mac).
Screen Shot 2019-05-30 at 8 59 27 AM

Reference:
Screen Shot 2019-05-30 at 8 58 42 AM

We do use the same delete key for removing a repository, but perhaps we meant to use the delete-left key? Can @desktop/engineering confirm?

**Comment by say25** [comment]
Left delete is

**Comment by shiftkey** [comment]
> Can @desktop/engineering confirm?

I'm honestly not fussed about what shortcut ends up being chosen here, but I'd like the reviewer(s) involved with this to test this on macOS and Windows and verify that:

 - the shortcut makes sense to the user
 - the shortcut does fire when the keyboard shortcut is pressed

Linux testing too would be desirable, but I suspect it'll overlap with how Windows handles this so it's not required.

**Comment by tierninho** [comment]
My only worry was that the average user has no idea what left- and right-delete are. 

Suggestion is we change both `Remove Repository` and `Discard all` to left-delete for simplicity. I assume this only affects Mac. @say25 ok to update your PR?

> I was mostly going for consistency. At the same time, CMD+Shift+Left Delete in Finder is empty trash maybe that makes more sense?

👍

**Comment by say25** [comment]
I don't mind making the tweak. If we are to change the shortcuts, should the `shift` modifier go with `Remove Repository` or `Discard All`?

**Comment by tierninho** [comment]
Thanks for helping. 

> should the shift modifier go with Remove Repository or Discard All?

"Discard all Changes" (Cmd/Ctrl+Shift+Delete) 
"Remove Repos" (Cmd/Ctrl+Delete)

**Comment by agisilaos** [comment]
While going through this PR I thought the same thing as @tierninho.

> My only worry was that the average user has no idea what left- and right-delete are.

**Comment by shiftkey** [comment]
@say25 tested this out on macOS and Windows and I'm happy with standardizing this on `CmdOrCtrl(+Shift)+Backspace` given the previous confusion discussed. 

I'll let this sit a couple of days and see if @desktop/engineering or @desktop/quality-assurance have any additional feedback or suggestions.

File: app/src/main-process/menu/build-default-menu.ts [hunk 1]
```diff
@@ -9,10 +9,8 @@ import { log } from '../log'
 import { openDirectorySafe } from '../shell'
 import { enableRebaseDialog, enableStashing } from '../../lib/feature-flag'
 import { MenuLabelsEvent } from '../../models/menu-labels'
+import { DefaultEditorLabel } from '../../ui/lib/context-menu'
```
Comment (AaronWashington) [inline app/src/main-process/menu/build-default-menu.ts]: https://github.com/enterprises/ENTERPRISE-NAME,

**Pull request merged**